cli/internal/helm/client.go

```diff
@@ -327,49 +325,19 @@ func (c *Client) applyMigrations(ctx context.Context, releaseName string, values
 	}
 
 	if currentV.Major == 2 && currentV.Minor == 7 {
```

Comment:
```suggestion
	if currentV.Major == 2 && currentV.Minor == 8 {
```
This branch should run if the current cluster is running v2.8